diff --git a/node_modules/macaroon/macaroon.js b/node_modules/macaroon/macaroon.js
index 0ade41d..8afabaa 100644
--- a/node_modules/macaroon/macaroon.js
+++ b/node_modules/macaroon/macaroon.js
@@ -72,6 +72,7 @@ const ByteBuffer = class ByteBuffer {
    */
   constructor(capacity) {
     this._buf = new Uint8Array(capacity);
+    this._capacity = capacity;
     this._length = 0;
   }
   /**
@@ -131,6 +132,7 @@ const ByteBuffer = class ByteBuffer {
     const newContent = new Uint8Array(newCap);
     newContent.set(this._buf.subarray(0, this._length));
     this._buf = newContent;
+    this._capacity = newCap;
   }
 };
 
